acctests/README.md

```diff
@@ -12,11 +10,10 @@ Here is the current process for deploying infrastructure that can be used for ru
 6. Download the vCenter Server appliance ISO image.
 
 ## Extract
-
 Next you need to mount and extract the contents of the iso to a location on your disk.
 
 ### macOS Users
-During phase 1 of the setup, macOS users may get security issues when ovftool is attempting to run, to prevent this issue please run:
+During the Equinix provisioning, macOS users may get security issues when ovftool is attempting to run, to prevent this issue please run:
```

Comment:
```suggestion
During the Equinix provisioning, macOS users may get security issues when ovftool is attempting to run, to prevent this issue run the following command:
```